src/libPMacc/PMaccConfig.cmake

```diff
@@ -29,7 +29,7 @@
 ###############################################################################
 # PMacc
 ###############################################################################
-cmake_minimum_required(VERSION 3.3)
+cmake_minimum_required(VERSION 3.0)
```

Comment:
Please remove this change from the pull request. There is no need to mix the cmake change with the new feature (TNG test)